templates/Dockerfile.apk.template

```diff
@@ -14,4 +14,10 @@ ENV RUNITDIR=/sbin
 
 COPY runit-init.sh /runit-init.sh
 
+# Alpine 3.21.* installs runit into a different directory. Test if this is 3.21
+# and then modify the above script appropriately.
+RUN source /etc/os-release && \
+    test "${VERSION_ID#3.21.}" = "$VERSION_ID" || \
```

Comment:
I know we have no way of knowing what future tags will do but would the idea to be to keep adding to this check any tags that also modify the location?

Reply:
Good question… alternately, we could assume future versions will keep this change, and only do this modification for lower versions.

Reply:
My intuition would be for this change to be maintained in future versions but perhaps we should let it fail again to ensure that is the case and if it is we can change it then.